### ISSUE TITLE
Multiple-table DELETE syntax error

### ISSUE THREAD

**Issue by ghost** [vitessio/vitess#3787]
Hey guys,

We found some problems when we test sql with Multiple-table:

```
Input: DELETE FROM a1, a2 USING t1 AS a1 INNER JOIN t2 AS a2 WHERE a1.id=a2.id;
Output: syntax error at position 16
```

But this sql will be fine:

```
DELETE a1, a2 FROM t1 AS a1 INNER JOIN t2 AS a2 WHERE a1.id=a2.id;
```

### [Multiple-Table Syntax](https://dev.mysql.com/doc/refman/5.7/en/delete.html)
```
DELETE [LOW_PRIORITY] [QUICK] [IGNORE]
    tbl_name[.*] [, tbl_name[.*]] ...
    FROM table_references
    [WHERE where_condition]

DELETE [LOW_PRIORITY] [QUICK] [IGNORE]
    FROM tbl_name[.*] [, tbl_name[.*]] ...
    USING table_references
    [WHERE where_condition]
```

**Comment by lvxin1986** [comment]
maybe u can execute the sql like this: DELETE a1, a2 FROM  t1 AS a1 INNER JOIN t2 AS a2 using(id)

**Comment by LepX** [comment]
感觉在这里遇到的中国人都是京东的哥们 。。

**Comment by aquarapid** [comment]
Fixed by #3872
